src/org/sosy_lab/java_smt/test/ModelEvaluationTest.java

```diff
@@ -144,4 +144,14 @@ public void testGetBooleansEvaluation() throws SolverException, InterruptedExcep
         Lists.newArrayList(null, false),
         Lists.newArrayList(null, bmgr.makeBoolean(false)));
   }
+
+  @Test
+  public void testGetStringsEvaluation() throws SolverException, InterruptedException {
+    requireStrings();
+    evaluateInModel(
+        smgr.equal(smgr.makeVariable("x"), smgr.makeString("hello")),
+        smgr.makeVariable("y"),
+        Lists.newArrayList(null, "hello"),
+        Lists.newArrayList(null, smgr.makeString("hello")));
+  }
 }
```

Comment:
This is currently the only test for String theory in this PR.
Would it be possible to provide a JUnit test for `StringFormulaManager` containing a few more (small) tests?